docs/JpManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -0,0 +1,213 @@
+# AWS での Scalar DL Auditor 導入方法
+
+本ガイドでは、Scalar DL Auditor を Amazon Web Services (AWS) のマネージド Kubernetes サービス (EKS) 上に手動でデプロイする方法を説明します。
+
+* Scalar DL Ledger は、改ざん検知可能なレコード (アセット) をデータベースに格納するコンポーネントです。詳細は、[Getting Started with Scalar DL](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started.md) のガイドを参照してください。また、AWS での Scalar DL Ledger の導入方法については[こちらのドキュメント](./docs/JpManualDeploymentGuideScalarDLOnAWS.md)を参照してください。
```

Comment:
Ditto.

Reply:
I fixed in 51c52000f99124aa1136d47a33150340b8aa82bc.